plugins/obs-filters/rnnoise/src/kiss_fft.c

```diff
@@ -39,7 +39,7 @@
 #define CUSTOM_MODES
 
 /* The guts header contains all the multiplication and addition macros that are defined for
-   complex numbers.  It also delares the kf_ internal functions.
+   complex numbers.  It also declares the kf_ internal functions.
```

Comment:
upstream fixed this 4 years ago: https://github.com/xiph/rnnoise/commit/0627990e0dc0b6de607a2575b7adb0efe67c181e

Reply:
Then it will be fixed when we update or unvendor this. I do not like patching vendored dependencies and I prefer only patching them when absolutely necessary because:
1. It can cause trivial patches to fail to apply cleanly.
2. You have to check that you aren't accidentally reverting your custom changes when you update.
3. It makes it harder to verify exactly which commit you vendored or what source you used.

Please remove the changes to the rnnoise files.